clappr/clappr-ios
# Add Background Layer

## 🏁 Goal

Create the new layers structure and add the Background Layer to it.

## ✅ How to test

1. Run the Clappr example app
2. Open the View Hierarchy Debugger
3. Check if views follow the image below:
![BackgroundLayer](https://user-images.githubusercontent.com/1668650/88221784-5934f100-cc3b-11ea-80dc-b294ec5fc339.jpg)
